Chaincase.UI/Services/Extensions.cs

```diff
@@ -30,6 +32,7 @@ public static IServiceCollection AddUIServices(this IServiceCollection services)
 
 		public static async Task NavigateBack(this NavigationManager navigationManager, IJSRuntime runtime, string route)
 		{
+			await runtime.InvokeVoidAsync("dispatchEventWrapper", "back");
```

Comment:
Navigate foward transitions are *chefs kiss* but clicking "< back" does nothing but disappear the button